charts/rancher-logging/rancher-logging/100.0.0+up3.10.0/templates/loggings/rke2/daemonset.yaml

```diff
@@ -10,13 +10,15 @@ spec:
       name: {{ .Release.Name }}-rke2-journald-aggregator
   template:
     metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/loggings/rke2/configmap.yaml") . | sha256sum }}
```

Comment:
Would this be easier as a secret? Or is that out of our control?

Reply:
I don't think a secret would be easier, this is just to force the fluent bit pods to redeploy when a new configuration is deployed.